ovirt-imageio/Dockerfile

```diff
@@ -1,16 +1,5 @@
-FROM quay.io/centos/centos:stream8
+FROM quay.io/ovirt/ovirt-imageio:latest
 
-RUN dnf install centos-release-ovirt45 -y
-RUN dnf install ovirt-imageio-daemon ovirt-imageio-common ovirt-imageio-client -y
+EXPOSE 80
 
-EXPOSE 12345
-
-COPY myticket.json myticket.json
-COPY 99-cdi.conf /etc/ovirt-imageio/conf.d/99-cdi.conf
 COPY cirros.raw /images/cirros.raw
-COPY server.key /tmp/certs/tls.key
-COPY imageio.crt /tmp/certs/tls.crt
-
-WORKDIR /
-
-#CMD ovirt-imageio& sleep 15 && ovirt-imageioctl add-ticket myticket.json && tail -n+1 -f /var/log/ovirt-imageio/daemon.log
```

Comment:
The `ovirt-imageio` image has an `ENTRYPOINT`, which can be combined with `CMD` to pass arguments.

So something like this should work I think:
```
CMD [ "/images/cirros.raw", "--ticket-id", "cirros" ]
```

Writing a command when running this image would overwrite the `CMD` options, but not the `ENTRYPOINT` launcher.